layouts/tv2_afvd_showstyle/Rundown_Header_Layout.json

```diff
@@ -45,14 +49,17 @@
 			"nextInCurrentPart": false,
 			"oneNextPerSourceLayer": false,
 			"plannedEndText": "",
-			"scale": 0.7,
+			"scale": 1,
 			"width": 4,
 			"height": 1,
 			"x": 8.8,
 			"y": 0.3,
 			"hidePlannedEnd": true,
 			"hideCountdown": true,
-			"xUnit": "%"
+			"xUnit": "%",
```

Comment:
The header is the primary example when panels' horizontal positioning has to be percentage-based because we want them to maintain proportion across multiple screen resolutions.